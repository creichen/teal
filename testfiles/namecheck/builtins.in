var int := 0;
type array[T] = array[T];
type string = string;
