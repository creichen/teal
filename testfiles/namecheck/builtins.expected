Multiple declarations for symbol 'array' at locations: 1:1-4:0, 2:1-2:18.
Multiple declarations for symbol 'int' at locations: 1:1-1:13, 1:1-4:0.
Multiple declarations for symbol 'string' at locations: 1:1-4:0, 3:1-3:21.
